src/helpers/api.js

```diff
@@ -1,6 +1,12 @@
 import axios from 'axios';
 import store from '@/store/index';
 import apiUrls from '@/setup/api-urls.json';
+import { i18n } from '@/setup/i18n';
+
+// DRY: Define axios instance with custom config applied to all requests.
+export const axiosInstance = axios.create({
+  locale: i18n.locale,
```

Comment:
todo: the position of the `locale` is invalid. You need to wrap it in the `headers` property. See https://axios-http.com/docs/instance